tests/ui/fmt/format-alignment-hash.rs

```diff
@@ -0,0 +1,5 @@
+fn main() {
+    // INVALID: alignment (`>18`) comes after `#X`, should be `:>#18X`
+    println!("{0:#X>18}", 12345);
```

Comment:
please move this into the [tests/ui/fmt/format-string-wrong-order.rs](https://github.com/rust-lang/rust/pull/145592/files#diff-ff158898027f15528d94c3fa0659d7baebc2a6ebd875127805d7d42afe5de14c) test